metisp/pymetis/src/pymetis/inputs/multiple.py

```diff
@@ -53,6 +53,20 @@ def __init__(self,
 
         self.extract_tag_parameters(tag_matches)
 
+    @classmethod
+    def getTargetName(cls, frameset: cpl.ui.FrameSet):  
```

Comment:
Nitpicking: please use snake case for function names in Python,
```suggestion
    def get_target_name(cls, frameset: cpl.ui.FrameSet):  
```

This is a simple enough solution I think for the multiple-target-recipes. We discussed other approaches, for example having 3 recipes: a generic one, and one for STD and one for SCI. But that is an orthogonal/complimentary solution to this one, so this is fine.

I'm not sure whether this is the best location for this function, but whatever.

Reply:
Agree,  I will change that.